tests/python/gaia-ui-tests/gaiatest/apps/notes/app.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base, PageRegion
+
+
+class Notes(Base):
+
+    name = 'Notes'
```

Comment:
When I removed `name = 'Notes'` the test failed to launch the notes application. When left in the test functioned properly and passed.